OracleWebCenterPortal/dockerfiles/12.2.1.4/README.md

```diff
@@ -153,7 +100,7 @@ $ docker container start -i WCPAdminContainer
 Run the following command to create the Portal Managed Server container:
 
 ```
-$ docker run -i -t --name WCPortalContainer --network=WCPortalNET -p <Any Free Port>:$MANAGED_SERVER_PORT -v $DATA_VOLUME:/u01/oracle/user_projects --env-file <directory>/webcenter.env.list $WCPortalImageName /bin/bash -c "/u01/oracle/container-scripts/configureOrStartWebCenterPortal.sh; /bin/bash"
+$ docker run -i -t --name WCPortalContainer --network=WCPortalNET -p <Any Free Port>:$MANAGED_SERVER_PORT -v $DATA_VOLUME:/u01/oracle/user_projects --env-file <directory>/webcenter.env.list $WCPortalImageName configureOrStartWebCenterPortal.sh
```

Comment:
You're still calling `configureOrStartWebCenterPortal.sh` manually here. This may not work on an orchestration platform like Kubernetes or Mesosphere. Can you not switch to using an environment variable to determine whether the container should start an admin server or the portal? Better yet, being able to automatically determine whether an admin server is already running would be great.